gradle/dependencies.gradle

```diff
@@ -108,19 +108,19 @@ versions += [
   lz4: "1.8.0",
   mavenArtifact: "3.9.6",
   metrics: "2.2.0",
-  mockito: "5.14.2",
-  opentelemetryProto: "1.0.0-alpha",
+  mockito: "5.18.0",
+  opentelemetryProto: "1.7.0-alpha",
```

Comment:
this is not a small update. Could you please revert it to keep the update "safe"?